src/System.Threading/tests/MutexTests.cs

```diff
@@ -70,11 +70,8 @@ public void Ctor_NameUsedByOtherSynchronizationPrimitive_Windows()
             }
         }
 
-        [Fact]
         [PlatformSpecific(TestPlatforms.Windows)]
-        [SkipOnTargetFramework(
-            TargetFrameworkMonikers.Uap,
-            "Impersonation APIs are not available and creating global sync objects is not allowed in UWP apps.")]
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWinRT))] // Can't create global objects in appcontainer
```

Comment:
UapNotUapAot is something different. IsWinRT/IsNotWinRT checks if you are inside an AppContainer. Uap/UapNotUapAot checks the targetframework but not the environment around it